applications/openshift/registry/ocp_secure_allowed_registries_for_import/rule.yml

```diff
@@ -0,0 +1,58 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: "Check configured allowed registries for import uses secure protocol"
+
+description: |-
+    The configuration <tt>allowedRegistriesForImport</tt> limits the container
+    image registries from which normal users may import images. This is a list
+    of the registries that can be trusted to contain valid images and the image
+    location configured is assumed to be secured unless configured otherwise. It
+    is important to allow only secure registries to avoid man in the middle attacks,
+    as the insecure image import request can be impersonated and could lead to
+    fetching malicious content.
+    List all the allowed repositories for import configured with insecure set to true
+    using the following command:
+    <pre>oc get image.config.openshift.io/cluster -o json | jq '.spec | (.allowedRegistriesForImport[])? | select(.insecure==true)'</pre>
+    Remove or edit the listed registries having insecure set by using the command:
+    <pre>oc edit image.config.openshift.io/cluster</pre>
+
+    For more information, follow
+    {{{ weblink(link="https://docs.openshift.com/container-platform/latest/openshift_images/image-configuration.html",
+                text="the relevant documentation") }}}.
+
+rationale: |-
+    Configured list of allowed registries for import should be from the secure source.
+
+severity: medium
+
+identifiers:
+  cce@ocp4: CCE-86235-9
+
+references:
+    nist: CM-7(2),CM-7(4),CM-7(5),CM-11
```

Comment:
same comments about the CM controls as the other rule

Reply:
comment incorporated.